File: Php/app/Urls/CashgameAddUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\HomegameUrlModel;
	use entities\Homegame;

	class CashgameAddUrlModel extends HomegameUrlModel{

		public function __construct(Homegame $homegame){
			parent::__construct(RouteFormats::cashgameAdd, $homegame);
		}

	}

}

File: Web/Models/CashgameAddUrlModel.cs
using Core.Classes;
using Web.Models;
using Web.Routing;

namespace app{

	public class CashgameAddUrlModel : HomegameUrlModel{

		public CashgameAddUrlModel(Homegame homegame) : base(RouteFormats.CashgameAdd, homegame){}

	}

}